main.tf

```diff
@@ -19,6 +19,10 @@ locals {
 
   # Get account ID
   account_id = ibm_resource_instance.en_instance.account_id
+
+  # Get public/private endpoint
```

Comment:
Can you mention its a temp workaround until https://github.ibm.com/Notification-Hub/adopters/issues/150 is implemented